doc/source/release-notes/Percona-XtraDB-Cluster-5.6.27-25.13.rst

```diff
@@ -0,0 +1,78 @@
+.. rn:: 5.6.27-25.13
+
+===================================
+Percona XtraDB Cluster 5.6.27-25.13 
+===================================
+
+Percona is glad to announce the release of |Percona XtraDB Cluster| 5.6.27-25.13 on January 11, 2015. Binaries are available from the `downloads section <http://www.percona.com/downloads/Percona-XtraDB-Cluster-56/release-5.6.27-25.13/>`_ or from our :ref:`software repositories <installation>`.
+
+Based on `Percona Server 5.6.27-75.0 <http://www.percona.com/doc/percona-server/5.6/release-notes/Percona-Server-5.6.27-75.0.html>`_, `Galera Replicator 3.13 <https://github.com/codership/galera/issues?q=milestone%3A25.3.13>`_ and on `Codership wsrep API 25.12 <https://github.com/codership/mysql-wsrep/issues?q=milestone%3A5.6.27-25.12>`_, Percona XtraDB Cluster 5.6.27-25.13 is now the current release. All Percona software is open-source and free. Details of this release can be found in the `5.6.27-25.13 milestone on Launchpad <https://launchpad.net/percona-xtradb-cluster/+milestone/5.6.27-25.13>`_.
```

Comment:
Right. It is based on 5.6.27.75.0 + 5.6.27-76.0

75.0 was to sync with MS - 5.6.27
76.0 was an incremental build done to accommodate some TokuDB changes as missed 75.0 train.

So mentioning both are important.